smssync/src/main/java/org/addhen/smssync/presentation/util/Utility.java

```diff
@@ -278,6 +278,9 @@ public static String getPhoneNumber(@NonNull Context context, @NonNull PrefsFact
      * @return String
      */
     public static String capitalizeFirstLetter(String text) {
+        if(text == null) return null;
```

Comment:
Actually your original code holds. However most of the code have braces on the if. This should do.

```java
if (text == null) {
    return null;
}
 
if (text.length() == 0) {
    return "";
}
 
if (text.length() == 1) {
    return text.toUpperCase();
}
return text.substring(0, 1).toUpperCase() + text.substring(1);

```
Thank you.